slam-libraries/viam-orb-slam3/orbslam_server_v1.cc

```diff
@@ -69,6 +69,47 @@ ::grpc::Status SLAMServiceImpl::GetPosition(ServerContext *context,
     return grpc::Status::OK;
 }
 
+::grpc::Status SLAMServiceImpl::GetPositionNew(
```

Comment:
I modeled this change following @jeremyhyde-viam 's lead here: https://github.com/viamrobotics/slam/pull/133

Reply:
Noice